padle/padle.py

```diff
@@ -0,0 +1,485 @@
+    #             prevTrans = mgraph.get_prev_transform(monster)
+    #             if(monster is not None and monster.name_en is not None and monster.on_na and
+    #                 ((monster.sell_mp >= 50000 and (monster.superawakening_count > 1 or prevTrans is not None)) or
+    #                  "Super Reincarnated" in monster.name_en) and not monster.is_equip and
+    #                     nextTrans is None and monster.level >= 99):
+    #                 final.append(str(i))
+    #         except Exception as e:
+    #             print("Error " + str(i))
+    #             pass
+    #     await ctx.tick()
+    #     print(",".join(final))
+
+    @padle.command()
+    async def guess(self, ctx, *, guess):
+        """Guess a card for the daily PADle"""
+        if ctx.guild is not None:
+            await ctx.send("You can only play PADle in DMs!")
+            return
+        if(not await self.config.user(ctx.author).start()):
+            await ctx.send("You have not started the game of PADle yet, try `^padle start`!")
+            return
+        if(await self.config.user(ctx.author).done()):
+            await ctx.send("You have already played today's PADle!")
+            return
+        dbcog = await self.get_dbcog()
+
+        guessMonster = await dbcog.find_monster(guess, ctx.author.id)
+        if guessMonster is None:
+            await ctx.send("Monster not found, please try again.", delete_after=10)
+            return False
+
+        monsterEmbed = EmbedView(
+            EmbedMain(
+                title=MonsterHeader.menu_title(guessMonster).to_markdown(),
+                description="Did you mean this monster?"),
+            embed_thumbnail=EmbedThumbnail(
+                MonsterImage.icon(guessMonster.monster_id))).to_embed()
+        confirmation = await self.get_embed_user_conf(ctx, monsterEmbed, timeout=20)
+        if not confirmation:
+            if confirmation is None:
+                await ctx.send("Confirmation timeout.", delete_after=10)
+            if confirmation is False:
+                await ctx.send("Please guess again.", delete_after=10)
+            return
+        if(not await self.config.user(ctx.author).start() and not await self.config.user(ctx.author).done()):
+            # confirming after day change or after finished
+            return
+        points = 0
+        async with self.config.user(ctx.author).guesses() as guesses:
+            guesses.append(guessMonster.monster_id)
+        monster = await dbcog.find_monster(str(await self.config.padleToday()), ctx.author.id)
+        nameLine = await self.getNameLine(ctx, monster, guessMonster)
+        otherLine = await self.getOtherLine(ctx, monster, guessMonster)
+        awakesLine = await self.getAwakesLine(ctx, monster, guessMonster)
+        data = [nameLine[0], otherLine[0], awakesLine[0]]
+        points += nameLine[1] + otherLine[1] + awakesLine[1]
+
+        async with self.config.user(ctx.author).embedText() as descr:
```

Comment:
this one would be a full menu2 implementation, but it's still a relatively straightforward one as far as menu2 goes, so see if you can do it, menu2 is a lot nicer than not menu2